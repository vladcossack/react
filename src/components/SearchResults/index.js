import React from 'react';
import { SearchResults, MoviesAmount } from '../styled.js';

const TotalMoviesAmount = ({ count }) => {
  return (
    <SearchResults className='container'>
      <MoviesAmount>{count} </MoviesAmount>
    movies found
    </SearchResults>
  );
};

export default TotalMoviesAmount;