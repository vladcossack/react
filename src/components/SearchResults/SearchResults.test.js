import React from 'react';
import rerender from 'react-test-renderer';
import SearchResults from '../SearchResults';

describe('when SearchResults', () => {

  test('then snapshot created', () => {
    const count = 200;
    const component = rerender.create(<SearchResults count={count}/>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});