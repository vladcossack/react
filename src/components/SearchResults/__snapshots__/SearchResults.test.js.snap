// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when SearchResults then snapshot created 1`] = `
<h3
  className="sc-fzqBZW gWWlHe container"
>
  <span
    className="sc-fzqNJr dayFnk"
  >
    200
     
  </span>
  movies found
</h3>
`;
